tests/e2e/main.go

```diff
@@ -56,6 +56,12 @@ var (
 			description: `This is like the happy path, but skips steps
 that involve starting or stopping nodes for the same chain outside of the chain setup or teardown.
 This is suited for CometMock+Gorelayer testing`,
+		},
+		"light-client-attack": {
+			testRun: DefaultTestRun(), steps: lightClientAttackSteps,
+			description: `This is like the short happy path, but uses light client attacks instead of double sign slashing.
```

Comment:
nit: maybe remove the `double sign slashing` wording for clarity, because as you have in the code above a light client attack could be an equivocation light client attack and hence `double sign`?

Reply:
Thanks, made the wording here a bit sharper to address this